lib/browser/define-property.ts

```diff
@@ -41,7 +38,8 @@ export function propertyPatch() {
   };
 
   Object.create = <any>function(obj: any, proto: any) {
-    if (typeof proto === OBJECT && !Object.isFrozen(proto)) {
+    // o is 'object' string
```

Comment:
`o`?

Reply:
yes, I have reverted this one, thank you!